### ISSUE TITLE
Missing "mypy" in tox envlist

### ISSUE THREAD

**Issue by mattdocumatt** [sphinx-contrib/cookiecutter#12]
I'm not too much skilled in tox but it looks that the "envlist" under "[tox]" section misses the mypy environment. Without it it is never tested.

https://github.com/sphinx-contrib/cookiecutter/blob/7e511df131c8f5d728504deb6cdc527adf14d947/%7B%7Bcookiecutter.project_name%7D%7D/tox.ini#L3

If you confirm it, I can save some of your time and fix it in simple PR.

**Comment by stephenfin** [comment]
Yes and no. The `envlist` lists the environments that run by default. You can still run this manually using `tox -e mypy`. We could add this, and I'll accept a PR that adds it, but it's not truly necessary
